apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableJobPostings.tsx

```diff
@@ -241,6 +243,16 @@ function ModalForm({
         >
           Add the job postings you want to publish to jobseekers at ReDI School.
         </Element>
+        {/* This Checkbox is added only for JobFair 2022. Please remove after 11.02.2022 */}
+        <Checkbox.Form
+          name="isJobFair2022JobListing"
+          checked={get(formik.values, 'isJobFair2022JobListing', false)}
+          handleChange={formik.handleChange}
+          {...formik}
+        >
+          Advertised at ReDI Job Fair 2022
```

Comment:
@lorepirri Can we change the text to "We will recruit for this job listing at the ReDI Job Fair on 11 February 2022" as in the ticket description? I think we overlooked that part.

Reply:
done, thanks @helloanil for spotting it,
I forgot to re-check the ticket description,
I've updated the screenshot of this PR as well.